src/main/java/securbank/models/Account.java

```diff
@@ -0,0 +1,205 @@
+package securbank.models;
+
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import javax.persistence.CascadeType;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+import org.hibernate.annotations.GenericGenerator;
+
+import org.joda.time.LocalDateTime;
+
+/**
+ * 
+ * 
+ * @author Madhu Illuri
+ *
+ */
+
+
+@Entity 
+@Table(name = "Account")
+public class Account {
+	
+	
+	/** map ManyToOne to many side of the relationship */
+	
+	@ManyToOne(cascade = CascadeType.ALL)
+	@JoinColumn(name = "userId", nullable = false)
+	private User user;
+	
+	/** Account number is unique. */
+	@Id
+	@NotNull
+	@GeneratedValue(generator = "uuid2")
+	@GenericGenerator(name = "uuid2", strategy = "uuid2")
+	@Size(min = 5, max = 15)
+	@Column(name = "accountNumber", nullable = false, length = 10, unique = true, columnDefinition = "BINARY(16)")
+	private UUID accountNumber;
+	
+	/** UserID is unique	 */
+	@NotNull
+	@Column(name = "userId", unique = true, nullable = false, columnDefinition = "BINARY(16)")
+	private UUID userId;
```

Comment:
@madhuilluri You have to create relationship with the user and not just a new column